adafruit_platformdetect/chip.py

```diff
@@ -103,6 +104,11 @@ def _linux_id(self): # pylint: disable=too-many-branches
             if compatible and 'amlogic, g12b' in compatible:
                 linux_id = S922X
 
+            cpu_model = self.detector.get_cpuinfo_field("cpu model")
+
+            if ("MIPS 24Kc") in cpu_model:
```

Comment:
It's failing checks because of the parenthesis here;

Reply:
I left it in parenthesis just in case there was something else to add on the detection. But I already fixed.

Thanks for reviewing it.

Reply:
Yeah, the linter like to remove excess stuff like that.